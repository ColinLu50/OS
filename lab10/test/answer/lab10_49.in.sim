2780
436
858
446
764
390
