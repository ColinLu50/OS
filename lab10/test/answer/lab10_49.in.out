FCFS
417 -> 352 -> 441 -> 47 -> 382 -> 149 -> 59 -> 113 -> 165 -> 200 -> 187 -> 108 -> 421 -> 412 -> 165 -> 175 -> 126 -> 305 -> 431 -> 380 -> 244 -> 279 -> 93
2780
SSTF
417 -> 421 -> 412 -> 431 -> 441 -> 382 -> 380 -> 352 -> 305 -> 279 -> 244 -> 200 -> 187 -> 175 -> 165 -> 165 -> 149 -> 126 -> 113 -> 108 -> 93 -> 59 -> 47
436
SCAN
417 -> 412 -> 382 -> 380 -> 352 -> 305 -> 279 -> 244 -> 200 -> 187 -> 175 -> 165 -> 149 -> 126 -> 113 -> 108 -> 93 -> 59 -> 47 -> 0 -> 421 -> 431 -> 441
858
C-SCAN
417 -> 412 -> 382 -> 380 -> 352 -> 305 -> 279 -> 244 -> 200 -> 187 -> 175 -> 165 -> 149 -> 126 -> 113 -> 108 -> 93 -> 59 -> 47 -> 0 -> 450 -> 441 -> 431 -> 421
446
LOOK
417 -> 412 -> 382 -> 380 -> 352 -> 305 -> 279 -> 244 -> 200 -> 187 -> 175 -> 165 -> 149 -> 126 -> 113 -> 108 -> 93 -> 59 -> 47 -> 421 -> 431 -> 441
764
C-LOOK
417 -> 412 -> 382 -> 380 -> 352 -> 305 -> 279 -> 244 -> 200 -> 187 -> 175 -> 165 -> 149 -> 126 -> 113 -> 108 -> 93 -> 59 -> 47 -> 441 -> 431 -> 421
390
